client/create.go

```diff
@@ -8,12 +8,22 @@ import (
 	"github.com/pinterest/knox"
 )
 
+func init() {
+	cmdCreate.Run = runCreate // break init cycle
+}
+
 var cmdCreate = &Command{
-	Run:       runCreate,
-	UsageLine: "create <key_identifier>",
+	UsageLine: "create [--key-template template_name] <key_identifier>",
 	Short:     "creates a new key",
 	Long: `
-Create will create a new key in knox with original data set as the primary data. Key data should be sent to stdin.
+Create will create a new key in knox with original data set as the primary data. There are two ways to provide data in order to create a new key.
+
+First way: key data is sent to stdin.
+Please use command "create <key_identifier>". 
```

Comment:
"Create will create a new key in knox with original data set as the primary data" -> is probably better as "Create will create a new key in knox with input as the primary key version"

You might want to reframe this as the default is to read from stdin unless the key-template option is given. Something like "Key data should be sent to stdin unless a key-template is specified".

Reply:
Thank you so much! I have revised my code according to your comment.